lib/src/video_compress/video_compressor.dart

```diff
@@ -124,8 +123,8 @@ extension Compress on IVideoCompress {
     String path, {
     VideoQuality quality = VideoQuality.DefaultQuality,
     bool deleteOrigin = false,
-    int? startTime,
-    int? duration,
+    double? startTime,
+    double? duration,
```

Comment:
The Dart's int is already 64-bit on all platforms (except web). No need to change it to a floating-point double type

https://stackoverflow.com/a/50429767